src/egl/main/egldisplay.c

```diff
@@ -238,7 +238,7 @@ _eglFindDisplay(_EGLPlatformType plat, void *plat_dpy)
          dpy->Platform = plat;
          dpy->PlatformDisplay = plat_dpy;
 
-         /* add to the display list */ 
+         /* add to the display list */
```

Comment:
Separate out the style fixes and submit upstream.

Reply:
Thanks @strassek, I have submitted the PR to upstream, https://gitlab.freedesktop.org/mesa/mesa/merge_requests/1200